src/test/java/com/google/sps/servlets/TestWeatherServlet.java

```diff
@@ -0,0 +1,131 @@
+package com.google.sps.servlets;
+
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.maps.model.LatLng;
+import com.google.sps.data.Weather;
+import com.google.sps.data.GeoCoding;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.Assert;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+public class TestWeatherServlet {
+  private static final double EPSILON = 0.0001;
+  private static final String RUSSIA_MOSCOW_ADDRESS = "Russia, Moscow, Red Square";
+  private static final LatLng RUSSIA_MOSCOW_LATLNG = new LatLng(55.75393030, 37.62079500);
+
+  @Mock
+  Weather weather;
+  @Mock
+  GeoCoding geoCoding;
+  @Mock
+  HttpServletRequest request;
+  @Mock
+  HttpServletResponse response;
```

Comment:
Indiciate in the name that this is a mock, e.g. rename weather to mockWeather, and so on.